web/src/lib/components/shared-components/navigation-bar/navigation-bar.svelte

```diff
@@ -30,7 +30,11 @@
   const logOut = async () => {
     resetSavedUser();
     const { data } = await api.authenticationApi.logout();
-    goto(data.redirectUri || '/auth/login?autoLaunch=0');
+    if (!data.redirectUri.startsWith('/')) {
+      window.location.href = data.redirectUri;
+    } else {
+      goto(data.redirectUri);
+    }
```

Comment:
Why is this necessary? Do we sometimes have an URI that's like `auth/login` and `goto` then does not understand that it's still the same host URL?

Reply:
Goto only works for internal urls now